tools/GraphClust/CollectResults/evaluation.py

```diff
@@ -4,59 +4,77 @@
 import re
 from sklearn import metrics
 from shutil import make_archive
+import sys
+import fnmatch, os
 
 def sh(script):
     system("bash -c '%s'" % script)
 
-dataNames = "FASTA/data.names"
+fasta_dir = sys.argv[1]
+results_dir = sys.argv[2]
+dataNames = fasta_dir+"/data.names"
 
 listOfClusters = []
 listOfHeaders = []
 headersNames = set()
-cluster_seqs_stats_path = "RESULTS/*.cluster.all"
-cluster_seqs_stats_files = glob.glob(cluster_seqs_stats_path)
+idsNames = set()
 
+
+names = os.listdir(results_dir)
+cluster_seqs_stats_files = fnmatch.filter(names, '*.cluster.all')
```

Comment:
@mmiladi please have a look at the [glob](https://docs.python.org/2/library/glob.html) module. I'm not sure if can sort the values, or if you really need them sorted, but glob might be useful for you in the future :)

Reply:
Of course, `glob` is the first choice, here it didn't work for me. It fails if directory name contains special character. E.g. with a downloaded file from galaxy `Galaxy282-[RESULTS.zip]`, [] are special characters.